main.nf

```diff
@@ -32,11 +32,11 @@ if(total_parameters_provided_by_user == 0) {
 
 // FAUST Required
 params.input_gating_set_directory = "$baseDir/input"
-params.active_channels_path = "$baseDir/helper_files/active_channels.rds"
-params.channel_bounds_path = "$baseDir/helper_files/channel_bounds.rds"
-params.supervised_list_path = "$baseDir/helper_files/supervised_list.rds"
 
 // FAUST Optional
+params.channel_bounds_path = ""
+params.active_channels_path = ""
+params.supervised_list_path = ""
```

Comment:
Very cool. Those were legacy defaults! Thank you for changing this.

Reply:
I was just lamenting the loss of these defaults and making some dummy RDS files as I transition all of my first-pass HIPCCyto runs to FAUST_Nextflow. So, great timing on this.